dkim/header_test.go

```diff
@@ -102,3 +102,11 @@ func TestHeaderPicker_Pick(t *testing.T) {
 
 	})
 }
+
+func TestFoldHeaderField(t *testing.T) {
+	header := `DKIM-Signature: a=rsa-sha256; bh=bvW0aiEWdP0ie2rawBb+IiTxlHI9KgEIYjEYTGzMRa0=; c=simple/simple; d=test.xxxxxxxxxx.xxxxx; h=Return-Path:From:To:Subject:Message-ID:Date; s=brisbane; t=1583859773; v=1; b=IIlaZ79pLDdypX2YzJuy9EvxsbrqF360CLhkQqMqVC/GIa1K/X8p8POny7WuVuFTMHLvA1zH9YcXgEgIH+X0GgGsqqzq8ETL+QGZ9g1EKCmIg+ZEYV5JnH25Ar6bJpGra3jg6sIxKi6CthpK50kd8T0Q3K/oZGot4BkLaPqogpo=;`
+	folded := foldHeaderField(header)
+	if strings.HasSuffix(folded, "\r\n \n") {
```

Comment:
Actually, can we just compare the result with the expected result? (Add a string including the `\r\n` sequences each 75 chars)

That way, if it breaks differently, we'll know too.